### ISSUE TITLE
label

### ISSUE THREAD

**Issue by chenhuiphd** [chaimi2013/3R#8]
Hi, Could you please tell me what the ground truth haze-free image is in a ND-Net network.  Thank you.

**Comment by chaimi2013** [comment]
> Hi, Could you please tell me what the ground truth haze-free image is in a ND-Net network. Thank you.

The ground truth should be the low-light image without color cast and haze. Note that you can use the NDR training set to train your model.
